tgui/packages/tgui/components/HealthStat.js

```diff
@@ -0,0 +1,28 @@
+import { classes } from 'common/react';
+import { COLORS } from "../constants";
+import { computeBoxClassName, Box } from './Box';
+/**
+* A box that applies a color to its contents depending on the damage type.
+* Accepted types: oxy, toxin, burn, brute.
+*/
+export const HealthStat = props => {
+  const {
+    type,
+    content,
+    className,
+    ...rest
+  } = props;
+  rest.color = COLORS.damageType[type] & COLORS.damageType[type];
+  return (
+    <Box
+      {...rest}
+      className={classes([
+        'HealthStat',
+        className,
+        computeBoxClassName(rest),
+      ])}
+      color={COLORS.damageType[type]}>
+      {content}
```

Comment:
Rather than using a `content` prop I think you can just use `children`, so usage would be like:
```
<HealthStat type="oxy">10</HealthStat>
```
Not requiring for this PR, would like to see applied in near future.